cmd/operator-sdk/generate/csv.go

```diff
@@ -306,3 +322,28 @@ func validateVersion(version string) error {
 	}
 	return nil
 }
+
+func (c *csvCmd) interactivePrompt() {
```

Comment:
Ideally the `Generator` would handle the interactive input using a library so the CLI doesn't have to know about CSV paths, type of input required, etc.